api/src/test/java/org/pmiops/workbench/api/OfflineClusterControllerTest.java

```diff
@@ -61,16 +54,20 @@ Clock clock() {
 
     @Bean
     public WorkbenchConfig workbenchConfig() {
-      WorkbenchConfig config = new WorkbenchConfig();
-      config.firecloud = new WorkbenchConfig.FireCloudConfig();
+      WorkbenchConfig config = WorkbenchConfig.createEmptyConfig();
+      config.firecloud = new FireCloudConfig();
       config.firecloud.clusterMaxAgeDays = (int) MAX_AGE.toDays();
       config.firecloud.clusterIdleMaxAgeDays = (int) IDLE_MAX_AGE.toDays();
       return config;
     }
   }
 
-  @Autowired ClusterApi clusterApi;
-  @Autowired OfflineClusterController controller;
+  @Qualifier(NotebooksConfig.SERVICE_CLUSTER_API)
+  @MockBean private ClusterApi mockClusterApi;
```

Comment:
I don't actually understand how this worked before. We were effectively treating it like a mock and it was happy, but it wasn't declared as a mock...

Reply:
It was declared as a mock...

Reply:
Oh I see. That makes sense.